src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -142,6 +143,42 @@ const AnalysisAdd = (
     <PageHeader style={{ paddingLeft: 20 }}>
       {I18n.t('musit.analysis.registeringAnalysis')}
     </PageHeader>
+    <Form horizontal style={{ paddingLeft: 10 }}>
+      <FormGroup>
+        <Label label="Type analyse" md={1} />
+        <Col md={2}>
+          <select
+            className="form-control"
+            onChange={updateFormField(form.analysisTypeCategory, updateForm)}
+          >
+            <option>Velg kategori</option>
+            {uniq(store.analysisTypes.map(b => b.category)).map(a => (
```

Comment:
The uniq filtering should not be done inside the component. This operation should be done before we send it in as props.

This comment is valid for all the other places you have used the select element.